### ISSUE TITLE
user shows 501 and group shows staff for archive.extracted

### ISSUE THREAD

**Issue by abhijo89** [saltstack/salt#25828]
I have a stat state which extractor the file to tmp folder. This is my code.

```
download_yii_framework:
  archive.extracted:
    - name: /tmp/
    - source: /home/test/test.tar
    - source_hash: md5={{ pillar['test_checksum'] }}
    - archive_format: tar
    - keep: False
    - if_missing: /tmp/test_tar/
    - user: root
    - group: root
```

After running this when i check the /tmp folder it shows 

```
drwxr-xr-x 5 501 staff 4096 Jun 30  test_tar
```

Why here user shows 501 and group shows staff ?

I am using salt 2015.5.0

**Comment by jfindlay** [comment]
@abhijo89, thanks for the report.  What user are you using to run the salt minion process on that system?

**Comment by cachedout** [comment]
@abhijo89 Those arguments are not supported in 2015.5.0. They will be available in the upcoming 2015.8, however.

**Comment by abhijo89** [comment]
I am running salt-minion as a service in sudo. 

@cachedout May i know,  how can i give user and group for extracted files and folders

**Comment by jfindlay** [comment]
@abhijo89, @cachedout is correct.  The `user` and `group` arguments were added in 2015.8.0, which should be coming out soon.

**Comment by abhijo89** [comment]
Ya. But how can i extract archive for a user.

**Comment by jfindlay** [comment]
@abhijo89, this is not implemented in the 2015.5 branch.  You are welcome to try 2015.8.0rc3 or you can recursively chown the files after extraction with a cmd.run.
